.travis.yml

```diff
@@ -5,6 +5,7 @@ jdk:
 branches:
   only:
     - master
+    - 7226-Closed-Email-Add-Premable-for-Instructors
```

Comment:
Done. I was aware that I should not include that change in the PR but it was done by mistake. Sorry for that.